docs/user-guide.md

```diff
@@ -304,7 +317,7 @@ Uploads user-provided build data to FOSSA. This allows users to manually provide
 ```bash
 # You can manually override the project and revision name.
 # This is useful when `fossa` can't manually infer them from `git`.
-FOSSA_API_KEY=YOUR_API_KEY_HERE fossa upload --project=PROJECT_NAME --revision=SOME_HASH --data=$(fossa analyze --no-upload --output)
+FOSSA_API_KEY=YOUR_API_KEY_HERE fossa upload --project=PROJECT_NAME --revision=SOME_HASH --data=$(fossa analyze --output)
```

Comment:
Why has this been changed? The code hasn't changed. This snippet will just upload twice.

Reply:
I changed the behavior to remove `--no-upload` and just have `--output`, which will output and exist.

Reply:
I don't see those changes, it doesn't look like `--output` exits.